spi/src/main/java/org/jboss/pnc/spi/BuildCoordinationStatus.java

```diff
@@ -31,6 +31,8 @@
 public enum BuildCoordinationStatus {
     NEW,
 
+    ENQUEUED,
```

Comment:
new state.
All tasks ready to build are given this state (the ones that have no dependencies or that have all the dependencies built)